src/bloom/utils.rs

```diff
@@ -455,6 +467,80 @@ impl BloomObject {
             _ => Err(BloomError::DecodeUnsupportedVersion),
         }
     }
+
+    /// This method is called from two different bloom commands BF.INFO and BF.INSERT. The functionality varies slightly on which command it
+    /// is called from. When called from BF.INFO this method is used to find the maximum possible size that the bloom object could scale to
+    /// without throwing an error. When called from BF.INSERT this method is used to determine if it is possible to reach the capacity that
+    /// has been provided.
+    ///
+    /// # Arguments
+    ///
+    /// * `capacity` - The size of the initial filter in the bloom object.
+    /// * `fp_rate` - the false positive rate for the bloom object
+    /// * `can_scale_to` - the capcity we check to see if it can scale to. If this method is called from BF.INFO this is set as -1 as we
+    ///                       want to check the maximum size we could scale up till
+    /// * `tightening_ratio` - The tightening ratio of the object
+    /// * `expansion` - The expanison rate of the object
+    ///
+    /// # Returns
+    /// * i64 - The maximum capacity that can be reached if called from BF.INFO. If called from BF.INSERT the size it reached after going past max capacity
```

Comment:
Does this sound right?
```suggestion
    /// * i64 - If called from BF.INFO, the maximum capacity that can be reached. If called from BF.INSERT, the size it reached before reaching the provided `can_scale_to`.
```

Reply:
It will be the capacity the moment it passes the `can_scale_to` i.e if we have `can_scale_to = 500` and starting capacity of 100 with expansion 2, we will return 700 (100 + 200 + 400). Maybe say the size it reached when it surpassed (or was greater than) the provided can_scale_to